lib/backpex/item_actions/item_action.ex

```diff
@@ -6,7 +6,7 @@ defmodule Backpex.ItemAction do
   @doc """
   Action icon
   """
-  @callback icon(assigns :: map()) :: %Phoenix.LiveView.Rendered{}
+  @callback icon(assigns :: map(), item :: struct() | nil) :: %Phoenix.LiveView.Rendered{}
```

Comment:
I thought about adding `item` to the `assigns` because `icon/1` is a component, but then it would not align to the `label/2` function 🤔 What do you think? @pehbehbeh

Reply:
I think it's OK the way it is here. Then pattern matching is also easy to understand.